infra-templates/ovh-dns/0/rancher-compose.yml

```diff
@@ -0,0 +1,49 @@
+# notemplating
+.catalog:
+  name: "OVH DNS"
+  version: "v0.1.0"
+  description: "Rancher External DNS service powered by OVH"
+  minimum_rancher_version: v1.6.0-rc1
+  questions:
+    - variable: "OVH_ENDPOINT"
+      label: "OVH Endpoint"
+      description: "Enter your endpoint"
+      type: "string"
+      required: true
+    - variable: "OVH_APPLICATION_KEY"
+      label: "OVH Application Key"
+      description: "Enter your application key"
+      type: "string"
+      required: true
+    - variable: "OVH_APPLICATION_SECRET"
+      label: "OVH Application Secret"
+      description: "Enter your application secret"
+      type: "string"
```

Comment:
Reviewing the code of ovh provider, `https://github.com/rancher/external-dns/blob/master/providers/ovh/ovh.go` line 43,  `OVH_APPLICATION_SECRET` is required. Could you please add it as required in rancher-compose.yml file?
```
+      required: true
```